community-operators/elasticsearch-operator/elasticsearch-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,217 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: elasticsearch-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Logging & Tracing"
+    capabilities: "Seamless Upgrades"
+    certified: "false"
+    description: |-
+      The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for tracing and cluster logging.
+    containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
+    createdAt: 2019-02-20T08:00:00Z
+    support: AOS Cluster Logging, Jaeger
+    alm-examples: |-
+        [
+            {
+                "apiVersion": "logging.openshift.io/v1alpha1",
+                "kind": "Elasticsearch",
+                "metadata": {
+                  "name": "elasticsearch"
+                },
+                "spec": {
+                  "managementState": "Managed",
+                  "nodeSpec": {
+                    "image": "quay.io/openshift/origin-logging-elasticsearch5:latest",
+                    "resources": {
+                      "limits": {
+                        "memory": "1Gi"
+                      },
+                      "requests": {
+                        "memory": "512Mi"
+                      }
+                    }
+                  },
+                  "nodes": [
+                    {
+                        "nodeCount": 1,
+                        "roles": ["client","data","master"], 
+                        "redundancyPolicy": "SingleRedundancy"
+                    }
+                  ]
+                }
+            }
+        ]
+spec:
+  displayName: Elasticsearch Operator
+
+  description: |
+    The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for use in tracing and cluster logging.
+    This operator only supports OKD Cluster Logging and Jaeger.  It is tightly coupled to each and is not currently capable of
+    being used as a general purpose manager of Elasticsearch clusters running on OKD.
+
+    Once installed, the operator provides the following features:
+    * **Create/Destroy**: Deploy an Elasticsearch cluster to the same namespace in which the Elasticsearch custom resource is created.
+
+  keywords: ['elasticsearch', 'jaeger']
+
+  maintainers:
+  - name: Red Hat, AOS Logging
+    email: aos-logging@redhat.com
+
+  provider:
+    name: Red Hat, Inc
+
+  links:
+  - name: Elastic
+    url: https://www.elastic.co/
+  - name: Elasticsearch Operator
+    url: https://github.com/openshift/elasticsearch-operator
+  installModes:
+  - type: OwnNamespace
+    supported: false
```

Comment:
OwnNamespace should be `true`
OLM outlines in their docs that OwnNamespace is for specifying whether the operator must watch it's own namespace, and with the AllNamespace InstallMode set to true this is implied to be the case.
OLM docs: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#operator-metadata

Reply:
@SamiSousa I'm missing why this is nees to be set to true if it implicitly includes it's own namespace.  "Implicit" suggests to me the value is immaterial.

Reply:
OwnNamespace is required for AllNamespaces because watching all namespaces will include watching the operator's namespace.
The OwnNamespace install mode represents the ability of the operator to watch CRs created in the operator's namespace when considering other install modes, and does not represent the ability of the operator to be installed in a single namespace. @alecmerdler

Reply:
updated to true